src/styles/index.scss

```diff
@@ -1,3 +1,82 @@
+@import "colors.scss";
+
+@import "stars.scss";
+
+@import "card.scss";
+
 body {
   margin: 0;
 }
+
+.code {
+  margin: 4px 84px 0 16px;
+  color: var(--gray-color);
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  word-wrap: break-word;
+}
+
+.rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin: 16px 16px 0;
+}
+
+.rewiev {
+  color: var(--text-color);
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  word-wrap: break-word;
+  margin: 0;
+}
+
+.mark {
+  margin: 24px 16px 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+}
+
+.price {
+  color: var(--gray-color);
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  word-wrap: break-word;
+  margin: 0;
+}
+
+.sum {
+  color: var(--text-color);
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  word-wrap: break-word;
+  margin: 0;
+}
+
+.buy {
+  margin: 16px;
+  height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. just use padding